### ISSUE TITLE
Locked warning is shown for all files in dvc status

### ISSUE THREAD

**Issue by prihoda** [iterative/dvc#927]
Hi there, the *locked file* warning seems to be shown for all files in dvc status:

```
git init
dvc init
git add .dvc 

# Create example folder
mkdir example
cd example

# Create hello.txt and world.txt
dvc run -o hello.txt 'yes "Hello" | head -10 > hello.txt'
dvc run -o world.txt 'yes "World" | head -10 > world.txt'

# Merge into hello_world.txt
dvc run -d hello.txt -d world.txt -o hello_world.txt 'paste hello.txt world.txt > hello_world.txt'

## Running command:
## 	yes "Hello" | head -10 > hello.txt
## Running command:
## 	yes "World" | head -10 > world.txt
## Running command:
## 	paste hello.txt world.txt > hello_world.txt

dvc status
## DVC file 'hello.txt.dvc' is locked. Its dependecies are not going to be shown in status output.
## DVC file 'world.txt.dvc' is locked. Its dependecies are not going to be shown in status output.
## DVC file 'hello_world.txt.dvc' is locked. Its dependecies are not going to be shown in status output.
## hello.txt.dvc
## world.txt.dvc

cat hello_world.txt.dvc 
## cmd: paste hello.txt world.txt > hello_world.txt
## deps:
## - md5: 2b360db687b5870bd328adbb56d3bf8f
##   path: hello.txt
## - md5: 53d23c8121564867f8ee222841a83811
##   path: world.txt
## md5: e1e60249c4f84c11cda133706d649e31
## outs:
## - cache: true
##   md5: 1abddce536be836e612dedc0a272b86f
##   metric: false
##   path: hello_world.txt
```

**Comment by prihoda** [comment]
Actually, it's shown for the non-locked files 😄 

The bug is in project.status:
```
        for stage in stages:
            if not stage.locked:
```

**Comment by efiop** [comment]
Oops, sorry :slightly_smiling_face: I'll fix it ASAP, will be included into 0.14.1 release today.

Thank you!
